integration-tests/cloud-wallet.test.ts

```diff
@@ -0,0 +1,181 @@
+import {IWallet} from '@docknetwork/wallet-sdk-core/lib/types';
+import {
+  closeWallet,
+  createNewWallet,
+} from './helpers/wallet-helpers';
+import {
+  DataStore,
+} from '@docknetwork/wallet-sdk-data-store/src/types';
+import {
+  SYNC_MARKER_TYPE,
+  initializeCloudWallet,
+} from '@docknetwork/wallet-sdk-core/src/cloud-wallet';
+import {createDataStore} from '@docknetwork/wallet-sdk-data-store-typeorm/src';
+
+const EDV_URL = process.env.EDV_URL;
+
+const agreementKey = {
```

Comment:
would it not be better to generate these keys the same way you will generate them for a wallet user?